templates/includes/records/hierarchy-full-item.html

```diff
@@ -15,10 +15,7 @@
 {% endif %}
 
 <div class="hierarchy-full-panel__bar-container-outer">
-    <div class="hierarchy-full-panel__bar-container-inner {% if is_current_item %} hierarchy-full-panel__bar-container-inner--active {% endif %} hierarchy-full-panel__bar-container-inner--level-{{forloop.counter|add:'1'}}">
-        <p><span class="hierarchy-full-panel__bar-number">N</span>
-        <span class="hierarchy-full-panel__bar-label">record(s)</span></p>
-    </div>
+    <div class="hierarchy-full-panel__bar-container-inner {% if is_current_item %} hierarchy-full-panel__bar-container-inner--active {% endif %} hierarchy-full-panel__bar-container-inner--level-{{forloop.counter|add:'1'}}"></div>
```

Comment:
```suggestion
    <div class="hierarchy-full-panel__bar-container-inner {% if is_current_item %} hierarchy-full-panel__bar-container-inner--active {% endif %} hierarchy-full-panel__bar-container-inner--level-{{ forloop.counter|add:'1' }}"></div>
```
small change space inside curly for readability.